vendors/arm/boards/musca-b1/aws_tests/config_files/aws_test_runner_config.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Amazon FreeRTOS V1.1.4
+ * Copyright (C) 2018 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ *
+ * http://aws.amazon.com/freertos
+ * http://www.FreeRTOS.org
+ */
+
+#ifndef AWS_TEST_RUNNER_CONFIG_H
+#define AWS_TEST_RUNNER_CONFIG_H
+
+/* Uncomment this line if you want to run AFQP tests only. */
+/* #define testrunnerAFQP_ENABLED */
+
+#define testrunnerUNSUPPORTED                      0
+
+/* Tests unsupported */
+#define testrunnerFULL_OTA_CBOR_ENABLED            testrunnerUNSUPPORTED
+#define testrunnerFULL_OTA_AGENT_ENABLED           testrunnerUNSUPPORTED
+#define testrunnerFULL_OTA_PAL_ENABLED             testrunnerUNSUPPORTED
+#define testrunnerFULL_MQTT_ALPN_ENABLED           testrunnerUNSUPPORTED
```

Comment:
It would be helpful to run at least one of the test groups that performs mutual authentication with AWS IoT. Realistically, that can find crypto port bugs that aren't yet caught by the PKCS11 tests.

Reply:
Yes. Actually we also plan to port the TCP/IP stack of Amazon FreeRTOS to arm platform Musca-B1 to connect to Amazon echo server. So that we can test the mutual authentication.